utils/execTo.js

```diff
@@ -0,0 +1,23 @@
+const {
+  execSync
+} = require(`child_process`);
+
+const execToStr = (cmd) => {
+  var _execSync;
+
+  return String((_execSync = execSync(cmd, {
+    encoding: `utf8`
+  })) !== null && _execSync !== void 0 ? _execSync : ``).trim();
+}
+
+const execToInt = (cmd) => {
+  // `parseInt` can return `NaN` for unexpected args
+  // `Number` can return undefined for unexpected args
+  // `0 | x` (bitwise or) will always return 0 for unexpected args, or 32bit int
+  return execToStr(cmd) | 0;
```

Comment:
We might want to use a ternary (or just a stock `if` / `else`) rather than a bitwise OR. `NaN`, `undefined` and `''` are treated as `false`.

![Screen Shot 2021-08-04 at 17 54 49](https://user-images.githubusercontent.com/1946433/128274382-10d1001c-3326-40d2-8173-114832d97ea6.png)

```js
  const resultString = execToStr(cmd);

  return resultString ? parseInt(resultString, 10) : 0;
```

Reply:
OG Typescript: will test 🤠